tests/http/test_session.py

```diff
@@ -0,0 +1,62 @@
+import random
+from unittest import mock
+
+import pytest
+import requests
+from requests import adapters
+from requests_mock import Mocker
+
+from up42.http import session as up42_session
+
+some_url = "https://something.com"
+
+methods_and_calls = [
+    ("GET", requests.Session.get),
+    ("POST", requests.Session.post),
+    ("PUT", requests.Session.put),
+    ("PATCH", requests.Session.patch),
+    ("DELETE", requests.Session.delete),
+    ("OPTIONS", requests.Session.options),
+    ("HEAD", requests.Session.head),
+]
+
+authorization_value = "Bearer some-token"
```

Comment:
since we are defining it outside the test, and using it at multiple places, this looks more like a global constant.
Consider renaming the variable to `AUTHORIZATION_VALUE`

Reply:
same comment for `some_url`